### DISCUSSION TITLE
Unable to track files through vendored inside sub-directory.

### DISCUSSION THREAD

**Discussion by ghost** [github-linguist/linguist#5409] [Classification] [answered]
### Why this line doesn't work : `static/fonts/**/* linguist-vendored=true` ?
> It is not ignoring files inside static/fonts/
> Is Syntax Wrong ?
Folder Structure of my project: _ProjectFolder_/static/fonts........(few more folders)

### .gitattribute file :

`text=auto`
`*.css linguist-vendored=true`
`*.scss linguist-vendored=true`
`static/fonts/**/* linguist-vendored=true`
`*.html linguist-vendored=true`
`*.js linguist-vendored=true`

**Comment by lildude** [comment, marked as answer]
Do you have a repo we can look at to see what is happening?

The reason I ask is two-fold:

1. your first sentence and the content of your `.gitattributes` don't match. The former should work, the latter won't because you're missing the final `*`.
2. I suspect you may be mixing up Search functionality (does `search` appear in your URL?) and Linguist functionality. Search is not influenced by `.gitattributes` options that hide files so you'll always be able to search for files but if your override is correct, they definitely won't count towards the language stats.

**Reply by ghost** [reply]
**Oh, Got it. An asterisk made all the difference.**
### Thanks a lot for your help.

**Reply by lildude** [reply]
>  2. I am not getting what you are talking about

🤔 so how are you determining the overrides are not taking effect? The `static/fonts/**/* linguist-vendored=true` override should do the trick.

You can see precisely what Linguist is detecting by installing Linguist locally and then run `github-linguist /path/to/repo --breakdown`. This will show you exactly what Linguist is doing on GitHub and will show you which files are responsible for each language.  If your override is working, then the files in `static/fonts/` will _not_ appear in the list.

You can use git directly to check your attributes completely independently of Linguist as I've demonstrated in https://github.com/github/linguist/discussions/5175#discussioncomment-317382

I'd normally do this myself, but can't as your repo is private. You could add me as a collaborator (I'm a GitHub employee) if you want.

**Reply by lildude** [reply]
Ooops, I didn't see your update come in whilst I typed out ☝️ 😄.
